src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-PostsPageCards/Components/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import * as S from '../styles';
+import {
+  Heading5,
+  Heading6,
+  Paragraph,
+} from '../../../ignitus-Atoms/typography';
+import { CardTypes, CardProps } from '../types';
+import { AppIcon } from '../../../../types/iconsTypes/iconEnums';
+/* eslint-disable react/destructuring-assignment */
+
+export const InternshipCard = ({
+  ImageLink,
+  Title,
+  Position,
+  Company,
+  Place,
+  JobType,
+}: CardTypes) => (
+  <Card>
+    <S.LeftRow>
+      <S.Avatar alt="profile" src={ImageLink} />
+    </S.LeftRow>
+    <S.Content>
+      <S.Row>
+        <Heading5 style={{ marginRight: '0.5rem' }}>{Title}</Heading5>{' '}
+        <Paragraph> has posted an internship</Paragraph>
+      </S.Row>
+      <S.Column>
+        <Paragraph>{Position}</Paragraph>
+        <Paragraph>{Company}</Paragraph>
+        <Paragraph>
+          {' '}
+          {JobType === 'remote' ? (
+            <S.Icon size="small" name={AppIcon.RemoteJobIcon} />
+          ) : (
+            <S.Icon size="small" name={AppIcon.LocationIcon} />
+          )}{' '}
+          {Place}
+        </Paragraph>
+      </S.Column>
+    </S.Content>
+  </Card>
+);
+
+export const PostsPageCards = ({
+  ImageLink,
+  Title,
+  Description,
+  type,
+}: CardProps) => (
+  <Card>
+    <S.LeftRow>
+      <S.Avatar alt="profile" src={ImageLink} />
+    </S.LeftRow>
+    <S.Content>
+      <S.Row>
+        <Heading5 style={{ marginRight: '0.5rem' }}>{Title}</Heading5>{' '}
+        {
+          {
+            publication: <Paragraph> has posted a publication</Paragraph>,
+            video: <Paragraph> has posted a video</Paragraph>,
+          }[type]
+        }
+      </S.Row>
+      <S.Row>
+        {
+          {
+            publication: <S.Icon name={AppIcon.PdfIcon} />,
```

Comment:
These icons look too big compared to text, just make it smaller. Also add some right margin to the icon, so that it looks distant from the text.